optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/employee/Employee.java

```diff
@@ -122,6 +123,7 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return 31 * ((31 * name.hashCode()) ^ contract.hashCode()) ^ skillProficiencySet.hashCode();
+        return 31 * ((31 * name.hashCode()) ^ contract.hashCode())
+                ^ skillProficiencySet.hashCode();
     }
```

Comment:
Do we still need this insanity when we no longer use GWT?
How about just `Objects.hash()`?